js/classes/entities/midground/npc/village/papa_chad.js

```diff
@@ -60,7 +60,7 @@ class PapaChad {
             this.velocity = new Vector(-Chad.DEFAULT_SPEED, 0);
         }
         // Add in gravity to the velocity.
-        this.velocity = Vector.add(this.velocity, new Vector(0, PHYSICS.GRAVITY_ACC));
+        this.velocity = Vector.add(this.velocity, Vector.multiply(new Vector(0, PHYSICS.GRAVITY_ACC), GAME.clockTick));
```

Comment:
Ohhh that's why he was falling through the ground right? Because his acceleration wasn't balanced by clocktick to make his falling consistent and making him phase through blocks?

Reply:
Yessir. He fell crazy fast. He's not *perfect* but hes usually there. If he's not, you can just try again by going to the village rq and back but it's pretty rare